cargo-dist/src/errors.rs

```diff
@@ -74,6 +74,10 @@ pub enum DistError {
     #[error(transparent)]
     TripleError(#[from] dist_schema::target_lexicon::ParseError),
 
+    /// error when using axoasset::toml::to_string() or similar
+    #[error(transparent)]
+    AxoassetTomlSerErr(#[from] axoasset::toml::ser::Error),
+
```

Comment:
This allows `do_migrate_from_v0()` to do `toml::to_string(&foo)?` and lean on miette.

Reply:
yeah it's chill to do transparent errors so long as we "own" the error somewhere and know it's good